WebService/appsettings.ini

```diff
@@ -23,6 +23,8 @@ iothub_data_folder = ./data/iothub/
 # and open connections.
 twin_read_write_enabled = "${?PCS_TWIN_READ_WRITE_ENABLED}"
 
+# The user-agent string is used to identify the application to the Azure IoT Hub
+user_agent = "remote_monitoring_device_simulation"
```

Comment:
How does this get set?  Would someone running DS underneath RM want this string while running it standalone would it be better to be device_simulation_service?  

Should Suite or the CLI set this similar to other env. variables?

Reply:
Yes, this value should be set from docker-compose file based on the solution type.

Reply:
I opted to hard-code the string in the appsettings.ini file to a RM specific string because so far as I know only RM uses this release, but you point about someone running simulation standalone is a good point.

I can also update the PR to check against the PCS_SOLUTION_TYPE env var to keep the tag as “devicesimulation” in case it is used by a Device Simulation deployment for some reason.